redis_session.c

```diff
@@ -231,7 +231,11 @@ PS_OPEN_FUNC(redis)
             if (url->query != NULL) {
                 array_init(&params);
 
+#if (PHP_MAJOR_VERSION == 7) && (PHP_MINOR_VERSION > 2)
```

Comment:
IMHO, using PHP_VERSION_ID is simpler (and will take care of PHP 8)

Reply:
Agree with @remicollet , something like `#if (PHP_VERSION_ID < 70300)` would be much better